.github/workflows/test.yml

```diff
@@ -33,11 +33,7 @@ jobs:
 
       - run: npm test
 
-      - run: npm run test-spec
-
-      - run: node ./bin.js trace test
```

Comment:
This is a different command. It runs the `trace` command to find dictionaries that contain the word `test`.  It just makes sure the command successful runs.